README.md

```diff
@@ -87,26 +87,27 @@ Before you install the plug-in, meet the following requirements:
 
```

Comment:
Edits for other parts of the ReadMe:

- Line 3: data sets should be two words
- Line 9: trim the sentence: "You can use the plug-in to perform various tasks:"
- Line 11: data sets two words; also ".. download **a** z/OS data set ..."
- Line 12: " ... to **a** z/OS **data set** or ..."
- Line 13: "... or download a job status and job spool files"
- Line 14: "Delete **a** z/OS **data set** ..."
- Line 15: "Submit **a** job from **a data set** ..."
- Line 16: "Make **a** USS directory"
- Line 17: "Allocate **a data set**"
- Line 19: lowercase: Software requirements
- Line 27: "**Ensure the** z/OS FTP service can be accessed remotely. Connect z/OS with the FTP client and run **the** command `rstat` in FTP client to check **that** option JESINTERFACELevel is set to 2."
- Line 29: Lowercase: Build the plug-in from source
- Line 33: What happens if this is not the first time? Should we add instructions for that scenario?
- Line 47: Same as Line 33.
- Line 86: Remove "Tip:." Tips and troubleshooting are two different things. This is troubleshooting, not a tip.

Reply:
> Line 27:

Reworded:
"Ensure the z/OS FTP service can be accessed remotely by connecting to z/OS from an FTP client. From the FTP client, run the command `rstat` to verify that the option `JESINTERFACELevel` is set to 2."


> Line 33: What happens if this is not the first time? Should we add instructions for that scenario?

Good point, we should advice people to run `npm install` after downloading  the source from GitHub 😋 

> Line 47

That sentence doesn't apply anymore, so I removed it 😋 

> Line 86:

Reworded:
"**Note:** When an unsuccessful message displays, you can troubleshoot the installation by addressing the issues that the message describes. For more details, you can review the logs located in the directory where you installed the Zowe CLI."